andela/pippin-ah-backend
# #163114828 Create connection model

#### What does this PR do?
This PR creates Model and migration files for the `follow` table

#### Description of Task to be completed?
- Create migrations file for the connection table
- Add models file for the `connection` table

#### How should this be manually tested?
To manually test this:
- Run `db:migrate`
- Sequelize should run migrations for the new `connection` table

#### What are the relevant pivotal tracker stories?
The relevant pivotal tracker story for this PR:
 - #163114828 https://www.pivotaltracker.com/n/projects/2229853/stories/163114828
